src/algorithms/telemetry_decoder/gnuradio_blocks/gps_l1_ca_telemetry_decoder_gs.cc

```diff
@@ -692,4 +692,4 @@ int gps_l1_ca_telemetry_decoder_gs::general_work(int noutput_items __attribute__
         }
 
     return 0;
-}
+}
```

Comment:
This change should not have been committed. Please figure out how to configure your editor or IDE to ensure that all files end in a newline.

There's no other change in this file.